p2p/CHANGELOG.md

```diff
@@ -7,8 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.3] - 2022-09-15
+
+**[TECHDEBT] AddrBook management optimization and refactoring** [#246](github.com/pokt-network/pocket/issues/246)
+
+- Added `peersManager` and `target` in order to abstract away and eliminate redundant computations
+- refactored debug logging in `getTarget` to print first and second target on the same line
+- Refactored AddPeerToAddrBook to use an event-driven approach in order to leverage sorted data structures
+- Added RemovePeerToAddrBook making use of the same pattern
+- Improved performance of AddPeerToAddrBook and RemovePeerToAddrBook by making the implementations O(n)
+- Updated stdnetwork to use a map instead of a slice
```

Comment:
```suggestion
- Added `peersManager` and `target` in order to abstract away and eliminate redundant computations
- Refactored debug logging in `getTarget` to print first and second target on the same line
- Refactored `AddPeerToAddrBook` to use an event-driven approach in order to leverage sorted data structures
- Added `RemovePeerToAddrBook` making use of the same pattern
- Improved performance of `AddPeerToAddrBook` and `RemovePeerToAddrBook` by making the implementations O(n)
- Updated `stdnetwork` to use a map instead of a slice
```